config/rbac/bases/api_manager_role.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: storageos:operator:api-manager
```

Comment:
Not, `storageos:api-manager` is created by api-manager. This role is about give roles to api-manager. So this belongs to operator, but because api-manager.